projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/utils.ts

```diff
@@ -2,3 +2,11 @@
  * Copyright 2021-2023 VMware, Inc.
  * SPDX-License-Identifier: Apache-2.0
  */
+export const TRANSFORM_JOB_TO_NOTEBOOK_BUTTON_LABEL =
+  'Transform Job To Notebook';
+export const RUN_JOB_BUTTON_LABEL = 'Run Job';
+export const CREATE_DEP_BUTTON_LABEL = 'Create Deployment';
+export const CREATE_JOB_BUTTON_LABEL = 'Create Job';
+export const RUN_FAILED_BUTTON_LABEL = 'Run Failed';

```

Comment:
Please double-check the codacy warnings if they are relevant, important before merging.